src/chi-vue/src/constants/types.ts

```diff
@@ -45,6 +45,7 @@ export interface DataTableRow {
   rowId: string;
   rowNumber: string;
   selected?: boolean;
+  disabledSelection?: boolean;
```

Comment:
```suggestion
  selectionDisabled?: boolean;
```
Thoughts on calling it the other way around?